### ISSUE TITLE
grpc version in Python

### ISSUE THREAD

**Issue by meawoppl** [grpc/grpc#17572]
There is no (obvious) way to determine the installed python version of grpc programmatically.
 
### What version of gRPC and what language are you using?
 
Lol
 
### What operating system (Linux, Windows, …) and version?
 
Windows/Linux
 
### What did you do?
If possible, provide a recipe for reproducing the error. Try being specific and include code snippets if helpful.
 
### What did you expect to see?
 
`grpc.__version__` / `grpc.get_version() -> tuple` or similar

**Comment by gnossen** [comment]
@meawoppl As of right now, we do not offer a public API for the version string. If you're willing to peek behind the curtain, you can use the following method to get access to it:

```python
>>> from grpc import _grpcio_metadata
>>> _grpcio_metadata.__version__
'1.17.1'
```

That's obviously not a long term solution for anyone. I'm working on a PR now to make this info available via `grpc.__version__`. If you don't mind my asking, what is your use case for this functionality?

**Comment by meawoppl** [comment]
We are seen performance impact on different versions and upgradsin our stack, and trying to track the variables that go into that.  Python packaging is pretty messy, so being able to capture or report the version string is valuable.

  We also want to have a optional call to chnanel.close() that only came in with release 1.12.0 or so, then send a warning about older versions, calling out the version string.

**Comment by gnossen** [comment]
https://github.com/grpc/grpc/pull/17580 was merged to master. You can now simply

```python
import grpc
print(grpc.__version__)
```

This functionality will be included in 1.18 very soon.
